shared-data/contrib/hints/hints/dragging-tags.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layouts/" + render_mode + "-tall.html" %}
+{% block title %}{{_("Dragging and dropping tags")}}{% endblock %}
+{% block content %}
+<div class="content-normal" style="max-width: 46em;">
+
+  <h1><span class="icon-tag"></span>
+    {{_("Dragging and dropping tags")}}
+  </h1>
+  <p>
```

Comment:
Is it worth adding a few words to explain the tag concept itself here? Not everyone is familiar with the concept; using synonyms ("labels", "categories") and explaining how tags differ from mailboxes?